planning/behavior_velocity_planner/src/scene_module/bus_stop/manager.cpp

```diff
@@ -0,0 +1,79 @@
+// Copyright 2020 Tier IV, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <lanelet2_extension/utility/query.hpp>
+#include <scene_module/bus_stop/manager.hpp>
+
+#include <tf2/utils.h>
+
+#include <memory>
+#include <set>
+#include <string>
+#include <unordered_map>
+#include <utility>
+#include <vector>
+
+namespace behavior_velocity_planner
+{
+namespace bus_stop
+{
+using lanelet::autoware::BusStop;
+
+BusStopModuleManager::BusStopModuleManager(rclcpp::Node & node)
+: SceneModuleManagerInterfaceWithRTC(node, getModuleName()), node_(node)
+{
+  const std::string ns(getModuleName());
+  planner_param_.buffer_size = node.declare_parameter(ns + ".buffer_size", 10);
```

Comment:
[imo] I think it is better not to define default value here since we can't notice it even if there is typo in config file. What would you think?

Reply:
@satoshi-ota 
Thanks for your review!
I think so too. I fixed.
https://github.com/tier4/autoware.universe/pull/339/commits/943ea987c96be8bce8ec068093ca99e3040e11a3

Reply:
Great :+1: